File: submissions/sm_014_kondreddy/week_02/day_03/session_02/count_vowels.js
var string="masaiSchool";
var i;
var j;
var vowels = ['a','e','i','o','u']
count = 0;
for (i = 0;i < string.length;i++) {
    for(j = 0;j < vowels.length;j++) {
    if (string[i] == vowels[j]) {
        count = count+1;
       // console.log(count)
        }
    }
}

console.log(count)

File: submissions/sm_102_amit/week_13/day_4/session_1/count_vowels.py
# Print the count of all the vowels in the given list of strings
strList = ["apple", "banana", "cherry"]
vowel = ["a", "e", "i", "o", "u"]
count = 0
for string in strList:
    for x in string:
        if x in vowel:
            count += 1
    print(string, count)
    count = 0
